assets/css/style.css

```diff
@@ -2234,3 +2234,12 @@ img.img-mouse {
 .team-v1 li > h3 {
 	color: #555;
 }
+
+/*Swagger Tables - Prevent Side Scrolling*/
+.parameter{
+  width: 100% !important;
+}
+
+.body-textarea {
+  width: 100% !important;
+}

```

Comment:
Well, now that I play with it, that only affects GET table bodies. Let's go with !important for now. But, can we move this into concur_assets/css/swagger-ui-styles.css, to keep things clean? Seems like it would be better suited in this css file.